skrf/network.py

```diff
@@ -1246,9 +1311,9 @@ def noisy(self):
       whether this network has noise
       """
       try:
-        return self.noise is not None and self.noise_freq is not None
+          return self.noise is not None and self.noise_freq is not None
       except:
-        return False
+          return False
 
```

Comment:
Yeah I definitely agree. Maybe @FournierVincent can chime in to see what he thinks. I think this would require moving some of the file import functionality over to NoisyNetwork. This would allow touchstone files with noise parameters to be imported and worked with.